kcb-ci-runner.sh

```diff
@@ -9,18 +9,8 @@ export RESET_KEYCLOAK=${RESET_KEYCLOAK:-true}
 export KEYCLOAK_STORAGE=${KEYCLOAK_STORAGE:-"JPA-Legacy-PostgreSQL"}
 if $RESET_KEYCLOAK; then
   cd $MINIKUBE_HOME
-  if [ "$KEYCLOAK_STORAGE" = "JPA-Map-PostgreSQL" ]; then
-    task reset-keycloak KC_STORAGE="jpa" KC_DATABASE="postgres"
-  elif [ "$KEYCLOAK_STORAGE" = "JPA-Legacy-PostgreSQL" ]; then
+  if [ "$KEYCLOAK_STORAGE" = "JPA-Legacy-PostgreSQL" ]; then
```

Comment:
```suggestion
  if [ "$KEYCLOAK_STORAGE" = "JPA-PostgreSQL" ]; then
```

I will adjust the Jenkins Job thats consuming this shell script.